src/maheshKJun20/StringExamples/SumOfDigitsInStringUsingException.java

```diff
@@ -0,0 +1,27 @@
+package maheshKJun20.StringExamples;
+
+public class SumOfDigitsInStringUsingException {
+	
+	void displaySumOfDigits(String str) {
+		System.out.println("Input: " + str );
+		String[] tempArray = str.split("");
+		String temp ="0";
+		int sum = 0;
+		for(int index=0;index<tempArray.length;index++) {
+			try {
+				temp += "" + Integer.parseInt(tempArray[index]);
+			}catch(NumberFormatException ne) {
+				sum += Integer.parseInt(temp);
+				temp = "0";
+			}
+		}
+		if(!temp.equals("0"))
+			sum += Integer.parseInt(temp);
+		System.out.println("Output: Sum of Digits in statement is: " + sum);
```

Comment:
String[] statementArray = statement.split(" ");
		int sum = 0;
		for (int index = 0; index < statementArray.length; index++) {
			try {
				sum += Integer.parseInt(statementArray[index]);
			} catch (NumberFormatException numberFormatException) {

			}
		}
		System.out.println("Input  : " + statement);
		System.out.println("Output : " + sum);

mahesh you have added many extra validation. which is not required.